packages/story-editor/src/components/videoTrim/useVideoNode.js

```diff
@@ -27,8 +32,10 @@ import { MEDIA_VIDEO_MINIMUM_DURATION } from '../../constants';
 function useVideoNode() {
   const [currentTime, setCurrentTime] = useState(0);
   const [startOffset, rawSetStartOffset] = useState(0);
-  const [maxOffset, setMaxOffset] = useState(Math.POSITIVE_INFINITY);
-  const [endOffset, rawSetEndOffset] = useState(Math.POSITIVE_INFINITY);
+  const [originalStartOffset, setOriginalStartOffset] = useState(0);
+  const [endOffset, rawSetEndOffset] = useState(100000);
+  const [originalEndOffset, setOriginalEndOffset] = useState(100000);
+  const [maxOffset, setMaxOffset] = useState(100000);
```

Comment:
Will `maxOffset` later be the length of the video in ms? And `originalEndOffset` be the offset of trimming relative to the original "parent" video?

Reply:
Yes, wasn't sure what to init these at. Could just be null. I originally had them at `Math.POSITIVE_INFINITY`, but `Math.POSITIVE_INFINITY !== Math.POSITIVE_INFINITY`, so `hasChanged` would be `true` for a split-second before `onmetadata` triggers.

I'll probably change all these to default to `null`.

Reply:
And yes, `maxOffset` is the same as the duration of the video in ms. `originalEndOffset` is, for now, the same as `maxOffset` but once we get to #8892 it might differ.

Reply:
Done in 03bb4c1798.